mlir/include/mlir/Dialect/AMX/Transforms.h

```diff
@@ -14,16 +14,20 @@ namespace mlir {
 class LLVMConversionTarget;
 class LLVMTypeConverter;
 class RewritePatternSet;
+class DialectRegistry;
 
 /// Collect a set of patterns to lower AMX ops to ops that map to LLVM
 /// intrinsics.
-void populateAMXLegalizeForLLVMExportPatterns(
-    const LLVMTypeConverter &converter, RewritePatternSet &patterns);
+void populateAMXLegalizeForLLVMExportPatterns(LLVMTypeConverter &converter,
```

Comment:
why not `const` any more?

Reply:
This is because I now add `amx::TileType` conversion into the passed converter in `populateAMXLegalizeForLLVMExportPatterns`. I've found a similar trick used for ARM SME extension in `populateArmSMEToLLVMConversionPatterns` method.